docs/marginaleffects.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>It would be nice to give a longer introduction motivating the problem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/473/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>